src/model/Service.php

```diff
@@ -1,37 +1,47 @@
 <?php
 namespace base\model;
 
+use base\model\entity\EntityFactory;
+use base\model\dataMapper\DataMapperFactory;
+
 class Service
 {	
 	protected $dataMapper;
 	protected $entityFactory;
-	protected $entityType;
 	
-	public function __construct($dataMapper,$entityType)
+	public function __construct($modelConfig)
     {
-		$this->dataMapper = $dataMapper;
-		$this->entityType = $entityType;
+		$dataMapperFactory = new DataMapperFactory($modelConfig["dataConnection"]);
```

Comment:
DataMapper is obviously link to dataConnection config, so if I need other keys from config, I will have to update this service? 
I dont want that, I will deeper explain my idea for a future feature